python/example_script.py

```diff
@@ -0,0 +1,59 @@
+"""An example Python script showing IAI Style Guide format and argparse usage.
+
+This Python script implements a dummy function that takes input and output file
+paths. The main purpose of this script is to show how a Python file should look
+in order to be presentable and conform to the IAI Python Style Guide. As a
+bonus, the usage of the argparse library is demonstrated for use with command-
+line interface (CLI).
+
+    Typical usage example in shell CLI:
+
+    python example_script.py -in ../foo/input.txt -out ../bar/output.txt
+
+:Author: Trond Linjordet
+"""
+
+import argparse
+
+def dummy_line_modification(line, i):
+    """Defines the modification to be perfomed for a single input line.
+
+    Args:
+        line: A string.
+        i: An integer.
```

Comment:
Once type hints are added types don't need to be mentioned here.

Reply:
Done. Note, type hints for keyword arguments did not work in a straightforward manner.